### PR TITLE
fix: add label for the impression data

### PR DESCRIPTION
**Pull request by ykhedher**

### CODE REVIEW

File: src/component/feature/FeatureForm/FeatureForm.tsx [hunk 1]
```diff
@@ -157,12 +160,20 @@ const FeatureForm: React.FC<IFeatureToggleForm> = ({
...
+                            control={
+                                <Switch
+                                    name="impressionData"
+                                    onChange={() =>
+                                        setImpressionData(!impressionData)
+                                    }
+                                    checked={impressionData}
+                                />
+                            }
+                            label="Impression Data"
```
Comment (thomasheartman) [inline src/component/feature/FeatureForm/FeatureForm.tsx:175]: Nice! 🥳

**Review by thomasheartman — CHANGES_REQUESTED** [review]
Nice one! 

Just a little thought. Now we have two labels that say "Impression Data" (sure, the top one is technically not a `label`, but it looks the same):

image

Do we want that? It feels a bit confusing.

How about changing the `Typography` element on lines 141-148 to a `FormControlLabel` and provide it a reference to the Impression data switch? That'll both cover the label duplication _and_ it feels more semantically correct to me (though admittedly, I may be off on this one) 💁🏼

**Comment by ykhedher** [comment]
[Impression data input is unlabeled](https://trello.com/c/Sd8Ubdfj/861-impression-data-input-is-unlabeled)

**Review by thomasheartman — APPROVED** [review]
Yeah, that's much better 😄 🙌🏼  Good work!

image

**Pull request merged**